pkg/config/protocol_config_test.go

```diff
@@ -135,6 +135,16 @@ func TestProtocolConfigurationValidation(t *testing.T) {
 		ValidatorsHistory: map[uint32]uint32{0: 1, 100: 4},
 	}
 	require.NoError(t, p.Validate())
+	p = &ProtocolConfiguration{
+		StandbyCommittee:  []string{},
+		CommitteeHistory:  map[uint32]uint32{0: 1, 100: 4},
+		ValidatorsHistory: map[uint32]uint32{0: 1, 100: 4},
+	}
+	require.Error(t, p.Validate(), "configuration should include StandbyCommittee")
```

Comment:
This won't check that error text contains "configuration should include StandbyCommittee" phrase, instead, it will print this string on `require` failure. You need to add one more line after this:
```suggestion
	err := p.Validate()
	require.Error(t, err)
	require.Contains(t, err.Error(), "configuration should include StandbyCommittee")
```